model/all.ttl

```diff
@@ -104,3 +105,36 @@ ref:TimeseriesReference a owl:Class,
             sh:nodeKind sh:IRIOrLiteral ;
             sh:path ref:storedAt ] .
 
+ref:VolttronReference a owl:Class,
+        sh:NodeShape ;
+    skos:definition "A reference to a point in a VOLTTRON instance" ;
+    rdfs:subClassOf ref:ExternalReference ;
+    sh:property [ 
+            a sh:PropertyShape ;
+            skos:definition "The topic name used by the platform agents for read and write processes" ;
+            sh:datatype xsd:string ;
+            sh:minCount 1 ;
+            sh:path ref:volttronTopicName 
+        ];
+    sh:property [ a sh:PropertyShape ;
+            skos:definition "Defines if the point is writable." ;
+            sh:datatype xsd:string ;
+            sh:minCount 1 ;
+            sh:maxCount 1 ;
+            sh:path ref:volttronIsWritable ;
+            sh:in (true false) ;
```

Comment:
Could this be `xsd:boolean` for the datatype instead? I think that matches the datatypes you have in `sh:in`